### PR TITLE
feat: design-singe-product-ui

### LINKED ISSUE #7: Design Single Product Page UI
- [x] Design details of single product as bellow:
> - [x] Product title.
> - [x] Product image.
> - [x] Prodcut price.
> - [x] Product description.
> - [x] Product category.
> - [x] Product rating(rate, count).
- [x] Design (Add to Card) button

### PR DESCRIPTION
**Pull request by messam10**

### CODE REVIEW

File: styles/SingleProduct.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    color: #ffffff;
+    padding: 8px 30px;
+    margin: 30px 0;
+    border-radius: 30px;
+    transition: background 0.5s;
+}
+
+.btn:hover {
+    background: #563434;
+}
```
Comment (Abrar-Abdulwahed) [inline styles/SingleProduct.css:35]: unused classes

File: styles/SingleProduct.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    padding-right: 25px;
+}
+
+.rating .fas {
+    color: #ff523b;
+}
+
+.rating .far {
+    color: #ff523b;
+}
```
Comment (Abrar-Abdulwahed) [inline styles/SingleProduct.css]: unused classes

File: src/components/SingleProduct.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+                    <div className="rating">
+                        <h5>{product.rating.rate}</h5>
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6">
+                            <path strokeLinecap="round" strokeLinejoin="round" d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z" />
+                        </svg>
+                    </div>
+
+                    <input type="number" defaultValue={1} />
+                    <a href="" className="btn">Add To Cart</a>
```
Comment (Abrar-Abdulwahed) [inline src/components/SingleProduct.jsx]: use component (Button)

**Review by Abrar-Abdulwahed — CHANGES_REQUESTED** [review]

File: styles/SingleProduct.css (hunk 1 above)
Comment (messam10) [reply styles/SingleProduct.css:35]: it's important to this code
`<Button className="btn" content={"Add To Cart"} />`

**Review by Abrar-Abdulwahed — APPROVED** [review]

**Pull request merged**